nats-server/subsz.yml

```diff
@@ -0,0 +1,58 @@
+type: json
+data:
+  url: /subsz
+  server: ${NATS_SERVER_HOSTPORT}
+
+observation:
+  - name: subsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: server.subscriptions.total
+        source: num_subscriptions
+        type: counter
+        label: subscriptions
+        description: The number of subscriptions
+
+      - name: server.subscriptions.inserts
+        source: num_inserts
+        type: counter
+        label: inserts
+        description: The number of insert operations in subscriptions list
+
+      - name: server.subscriptions.removes
+        source: num_removes
+        type: counter
+        label: removes
+        description: The number of remove operations in subscriptions list
+
+      - name: server.subscriptions.matches
+        source: num_matches
+        type: counter
+        label: matches
+        description: The number of times a match is found for a subscription
+
+      - name: server.cache.size
+        source: num_cache
+        type: long_gauge
+        label: cache size
+        description: The number of result sets in the cache
+
+      - name: server.cache.hit.rate
+        source: cache_hit_rate
+        type: double_gauge
+        label: cache hit rate
+        description: The rate matches are being retrieved from cache (value is in [0-1])
```

Comment:
enclose this description with-in double quotes

Reply:
DONE